src/command_modules/azure-cli-shell/azclishell/app.py

```diff
@@ -401,12 +401,15 @@ def _special_cases(self, text, cmd, outside):
 
         if cmd.strip() == "quit" or cmd.strip() == "exit":
             break_flag = True
-        elif text.strip() == "clear":  # clears the history, but only when you restart
+        elif text.strip() == "clear-history":  # clears the history, but only when you restart
             outside = True
             cmd = 'echo -n "" >' +\
                 os.path.join(
                     SHELL_CONFIG_DIR(),
                     SHELL_CONFIGURATION.get_history())
+        elif text.strip() == 'clear':
+            outside = True
+            cmd = 'clear'
```

Comment:
It's fine to map clear to cls on CMD if you want to--just make sure it works!  In this case, "clear" would be a shell-supported keyword (like exit, quit and clear-history).  The alternative would be to just rely on your existing # syntax, so bash users would type `#clear` and CMD users would type `#cls`.

As a crazy CMD user, when I'm not in shell I have to type cls, so it would be a little weird for me to have to remember to use clear when in the shell. For me, #cls would be more intuitive.